### PR TITLE
Add value to checkbox property editor

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
Belongs to https://github.com/umbraco/UmbracoDocs/issues/2714

### CODE REVIEW

**Comment by sofietoft** [comment]
Looks great @erikjanwestendorp !

Thanks for adding this 💪 
I'll make sure it is reviewed as soon as possible!

File: Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/True-False/index.md [hunk 1]
```diff
@@ -22,11 +22,37 @@ It is also possible to define a label, that will be displayed next to the checkb
...
+```
+
+## Add value
+
+See the example below to see how a value can be added or changed programmatically. To update a value of a property editor you need the [Content Service](../../../../../Reference/Management/Services/ContentService/index.md).
+
+```csharp
+var contentService = Services.ContentService;
+
+var content = _contentService.GetById(1234); // ID of your page
```
Comment (sofietoft) [inline Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/True-False/index.md:54]:
```suggestion
var content = contentService.GetById(1234); // ID of your page
```

**Review by sofietoft — CHANGES_REQUESTED** [review]
The code samples looks great @erikjanwestendorp !

Would you be up for adding the GUID samples, like it's been done here as well: https://our.umbraco.com/documentation/Reference/Management/Services/ContentTypeService/Retrieving-content-types

**Comment by erikjanwestendorp** [comment]
@sofietoft Thanks for the suggestion. I Just committed an example with the use of a GUID

**Review by sofietoft — APPROVED** [review]
Looks great @erikjanwestendorp !

Thanks for adding the GUID in there as well, and for showing both ways to do it 💪 
Much appreciated.
Let's get this merged in then 😄

**Pull request merged**